src/main/java/com/internet/shop/controllers/order/CompleteOrderController.java

```diff
@@ -0,0 +1,30 @@
+package com.internet.shop.controllers.order;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.interfaces.OrderService;
+import com.internet.shop.service.interfaces.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CompleteOrderController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+    private OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        Order order = orderService.completeOrder(shoppingCart);
+        order.setUserId(shoppingCart.getUserId());
```

Comment:
isn't it done already inside completeOrder()?

Reply:
yes, no need to create order